tests/CommandHintsTest.php

```
@@ -0,0 +1,65 @@
+	/**
+	 * Collect the hints declared within a fixture directory.
+	 *
+	 * @param string $fixture Name of the directory within `tests/data`.
+	 * @return array<string, string> Map of full command name to hint.
+	 */
+	private function get_hints_from_fixture( string $fixture ): array {
+		$class  = new ReflectionClass( CommandHints::class );
+		$method = $class->getMethod( 'get_hints_from_vendor_dir' );
+		if ( PHP_VERSION_ID < 80100 ) {
+			// @phpstan-ignore method.deprecated
+			$method->setAccessible( true );
```

PHPStan (2.2.8)

[error] No error with identifier method.deprecated is reported on line 24.


(ignore.unmatchedIdentifier)